.env.production

```
@@ -0,0 +1,23 @@
+# Production Environment Variables for Render
+# Copy these to your Render Environment tab
+
+# Supabase Database (replace [PROJECT-REF] with your actual project reference)
+DATABASE_URL=postgresql://postgres.[PROJECT-REF]:kybVax-6nesce-vovfoq@aws-0-us-west-1.pooler.supabase.com:6543/postgres
+
+# ClickUp Integration (keep your existing values)
+CLICKUP_TOKEN=pk_132000182_VEWS6RMM30M2794Q20J7HEWG9IOP2L7M
+CLICKUP_TEAM_ID=9013769810
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CLICKUP_TEAM_ID key should go before the CLICKUP_TOKEN key

(UnorderedKey)